src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommand.cs

```diff
@@ -53,47 +53,63 @@ private string[] TransformArgs()
 
             if (_appliedCommand.HasOption("include-prerelease"))
             {
-                CheckForOutdatedOrDeprecated("--include-prerelease");
+                CheckForOutdatedOption("--include-prerelease");
             }
 
             if (_appliedCommand.HasOption("highest-patch"))
             {
-                CheckForOutdatedOrDeprecated("--highest-patch");
+                CheckForOutdatedOption("--highest-patch");
             }
 
             if (_appliedCommand.HasOption("highest-minor"))
             {
-                CheckForOutdatedOrDeprecated("--highest-minor");
+                CheckForOutdatedOption("--highest-minor");
             }
 
             if (_appliedCommand.HasOption("config"))
             {
-                CheckForOutdatedOrDeprecated("--config");
+                CheckForOnlineOptions("--config");
             }
 
             if (_appliedCommand.HasOption("source"))
             {
-                CheckForOutdatedOrDeprecated("--source");
+                CheckForOnlineOptions("--source");
             }
 
-            if (_appliedCommand.HasOption("deprecated") && _appliedCommand.HasOption("outdated"))
+            var mutexOptionCount = 0;
+            mutexOptionCount += _appliedCommand.HasOption("deprecated") ? 1 : 0;
+            mutexOptionCount += _appliedCommand.HasOption("outdated") ? 1 : 0;
+            mutexOptionCount += _appliedCommand.HasOption("vulnerable") ? 1 : 0;
+            if (mutexOptionCount > 1)
             {
-                throw new GracefulException(LocalizableStrings.OutdatedAndDeprecatedOptionsCannotBeCombined);
+                throw new GracefulException(LocalizableStrings.OptionsCannotBeCombined);
             }
 
             return args.ToArray();
         }
 
         /// <summary>
-        /// A check for the outdated and deprecated specific options. If --outdated or --deprecated not present,
+        /// A check for options requiring online services. If no online options are present,
         /// these options must not be used, so error is thrown.
         /// </summary>
         /// <param name="option"></param>
-        private void CheckForOutdatedOrDeprecated(string option)
+        private void CheckForOnlineOptions(string option)
```

Comment:
nit: what does "Online" mean

Reply:
Options that require online lookups to services for their information

Reply:
(code removed per main comment above)